pallets/runtime/tests/src/relayer_test.rs

```diff
@@ -0,0 +1,128 @@
+use super::{
+    storage::{get_last_auth_id, TestStorage, User},
+    ExtBuilder,
+};
+use frame_support::{assert_noop, assert_ok, StorageMap};
+use polymesh_common_utilities::traits::transaction_payment::CddAndFeeDetails;
+use polymesh_common_utilities::Context;
+use polymesh_primitives::{Signatory, TransactionError};
+use polymesh_runtime_develop::{fee_details::CddHandler, runtime::Call};
+use sp_runtime::transaction_validity::InvalidTransaction;
+use test_client::AccountKeyring;
+
+type Relayer = pallet_relayer::Module<TestStorage>;
+type Identity = pallet_identity::Module<TestStorage>;
+type AccountKeyRefCount = pallet_identity::AccountKeyRefCount<TestStorage>;
+
+type Error = pallet_relayer::Error<TestStorage>;
+
+// Relayer Test Helper functions
+// =======================================
+
+macro_rules! assert_key_usage {
+    ($user:expr, $usage:expr) => {
+        assert_eq!(AccountKeyRefCount::get(&$user.acc()), $usage);
+    };
+}
```

Comment:
This can just be a function and you can use `#[track_caller]` for the macro-like location.